package.json

```diff
@@ -42,6 +42,7 @@
     "@sparing-software/commitlint-config": "^1.0.0",
     "@sparing-software/prettier-config": "^1.0.0",
     "@types/node": "^18.16.0",
+    "axios": "^1.7.4",
```

Comment:
A czy axios musi być? A jak już to nie powinien być jako peer? Chyba że on faktycznie jest używany tylko na potrzeby tej paczki

Reply:
@adrianklimek Jest jako dev dependency, potrzebny tylko do testów. Zainstalowany żeby rozwiązać ten problem https://mattermost.sparing.cloud/sparing/pl/jmuohkkxo7rpbjfygzmohxbkie
Próbowałem też jako alternatywę zastosować drugie podejście opisane tu https://mattermost.sparing.cloud/sparing/pl/6wxshz83riy67rpx6owmxc5m1o
Niestety, nie udało mi się sprawić żeby plik dało się importować i żeby też został nieprzetworzony w wygenerowanej paczce. Proponowane rozwiązanie jakie znalazłem to było kopiowanie go przy buildzie